.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Ideally this ignore would be in your global ignore file, not in a project ignore file.